cdap-master/src/test/java/co/cask/cdap/master/startup/ConfigurationCheckTest.java

```diff
@@ -62,4 +62,9 @@ public void invalidAuditTopicTest() {
   public void invalidLogPartition() {
     runConfigurationCheck(Constants.Logging.LOG_PUBLISH_PARTITION_KEY, "application1");
   }
+
+  @Test (expected = RuntimeException.class)
+  public void invalidHBaseDDLExecutorConfTest() {
```

Comment:
it checks for the no ext dir case as it check against `cdap-default.xml`, right? Or do we already have a test that validates by setting the ext dir, an extension is being picked up correct?